src/Afsy/Bundle/FrontBundle/Controller/AventController.php

```diff
@@ -40,8 +40,9 @@ class AventController extends Controller
             '01-presentation-afsy'                                        => 'AfsyFrontBundle:Avent:day_2013_01.html.twig',
         ),
         2017 => array(
-            '02-la-nouvelle-configuration-par-defaut-de-symfony4' => 'AfsyFrontBundle:Avent:day_2017_02.html.twig',
             '01-easyadminbundle-l-arriere-guichet-easy-peasy'     => 'AfsyFrontBundle:Avent:day_2017_01.html.twig',
+            '02-la-nouvelle-configuration-par-defaut-de-symfony4' => 'AfsyFrontBundle:Avent:day_2017_02.html.twig',
+            '03-deployer-un-projet-symfony-flex-sur-heroku'       => 'AfsyFrontBundle:Avent:day_2017_03.html.twig',
```

Comment:
It's important to sort them by decreasing order to make the most resent post appear first.

Reply:
Oh I thought it was the contrary 😄